Documentation/learning/design-auth-v3.md

```diff
@@ -1,5 +1,7 @@
 ﻿---
 title: etcd v3 authentication design
+weight: 3500
+description: Differences between etcd v3 authentication & v2 authentication

```

Comment:
Nit, we should keep description focusing on just v3 design (as the doc is not only focused on v2 vs v3), may be just "etcd v3 authentication " if that sounds good? Thanks!

Reply:
Updated